includes/Modules/Search_Console.php

```diff
@@ -371,14 +371,28 @@ protected function create_search_analytics_data_request( array $args = array() )
 		if ( ! empty( $args['end_date'] ) ) {
 			$request->setEndDate( $args['end_date'] );
 		}
+
+		// Only include data for URLs that are part of the current site.
+		$filter = new Google_Service_Webmasters_ApiDimensionFilter();
```

Comment:
Although slightly more verbose, I think it would be preferable to name the filter variables according to the filter they apply for additional clarity - particularly because the same variable is currently used for both cases.
Something like `$scope_site_filter` for the first site-wide filter, and maybe `$scope_url_filter` for the page-specific one would be more clear?